cli/hzn.go

```diff
@@ -727,6 +728,10 @@ Environment Variables:
 			// get HZN_EXCHANGE_USER_AUTH as default if exUserPw is empty
 			exUserPw = cliutils.RequiredWithDefaultEnvVar(exUserPw, "HZN_EXCHANGE_USER_AUTH", msgPrinter.Sprintf("exchange user authentication must be specified with either the -u flag or HZN_EXCHANGE_USER_AUTH"))
 		}
+
+		if err := version.VerifyExchangeVersion1(exchange.LoadExchangeVersion(*exOrg, *exUserPw), false, true); err != nil {
```

Comment:
Some commands use -n instead of -u for cred. So the code here should use credToUse if it is not empty, otherwise use exUserPw.

Reply:
Implementation has been changed. Please look into, does it look OK now?